docs/en/observability/quickstarts/monitor-k8s-otel.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[monitor-k8s-otel-edot]]
+= Quickstart: Unified Kubernetes Observability with Elastic Distributions of OpenTelemetry (EDOT)
+
+preview::[]
+
+In this quickstart guide, you will learn how to send Kubernetes logs, metrics, and application traces to Elasticsearch using the https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator] to orchestrate the https://github.com/elastic/opentelemetry/tree/main[Elastic Distributions of OpenTelemetry] (EDOT) Collectors and SDK instances you will need.
+
+All the components will be deployed through the https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack[opentelemetry-kube-stack] helm chart. They include:
+
+* https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator]
+* `DaemonSet` EDOT Collector configured for node level metrics.
+* `Deployment` EDOT Collector configured for cluster level metrics.
+* Instrumentation object for applications https://opentelemetry.io/docs/kubernetes/operator/automatic/[auto-instrumentation].
+
+For a more detailed description of the components and advanced configuration refer to the https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md[elastic/opentelemetry] GitHub repository.
+
+[discrete]
+== Prerequisites
+
+* An {es} cluster, self-managed or a hosted {ess} deployment on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data.
+* A running Kubernetes cluster (v1.23 or newer).
+* https://kubernetes.io/docs/reference/kubectl/[Kubectl].
+* https://helm.sh/docs/intro/install/[Helm].
+* (optional) https://cert-manager.io/docs/installation/[Cert-manager], if you opt for automatic generation and renewal of publicly trusted TLS certificates.
+
+[discrete]
+== Collect your data
+
+.  In {kib}, go to **Observability** and click **Add Data**.
+
+. Under the question *`What do you want to monitor?`* select **Kubernetes**, and then select the **OpenTelemetry: Full Observability** option.
++
+[role="screenshot"]
+image::images/quickstart-k8s-otel-entry-point.png[Kubernetes-OTel entry point]
+
+. Follow the on-screen instructions to perform the installation of all needed components.
++
+[NOTE]
+====
+The default installation deploys the OpenTelemetry Operator with a self-signed TLS certificate. For automatic certificate renewal with https://cert-manager.io/docs/installation/[cert-manager], refer to https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md#cert-manager[cert-manager integrated installation] for instructions on customizing the `values.yaml` file before running the `helm install` command.
```

Comment:
This sentence also feels a little long. Maybe we can break it up. Something like:

The default installation deploys the OpenTelemetry Operator with a self-signed TLS certificate. Refer to the cert-manager integrated installation guide to enable automatic certificate renewal using cert-manager. This guide provides instructions on customizing the values.yaml file before running the helm install command.

Reply:
Thanks! It looks much better... I have actually removed the final part `This guide provides instructions on customizing the values.yaml file before running the helm install command.`, as it's probably unneeded. Let me know your thoughts. It will look like:

> The default installation deploys the OpenTelemetry Operator with a self-signed TLS certificate. Refer to the https://github.com/elastic/opentelemetry/blob/main/docs/kubernetes/operator/README.md#cert-manager[cert-manager integrated installation] guide to enable automatic certificate renewal using https://cert-manager.io/docs/installation/[cert-manager].

Reply:
@bmorelli25 , I had to update a bit the info about the TLS certificate in a last minute commit, as cert-manager in this case is used for automatic generation (AND RENEWAL) of the certificate (still self-signed but automatically renewed).
The information about publicly trusted certs was incorrect, although that's a very common use case for `cert-manager`.

would you please review?

Reply:
This edit looks good too!